jax_galsim/core/wrap_image.py

```diff
@@ -0,0 +1,35 @@
+# from functools import partial
+import jax
+import jax.numpy as jnp
+
+
+@jax.jit
+def wrap_nonhermition(im, xmin, ymin, nxwrap, nywrap):
```

Comment:
Not too familiar with wrapping myself, so @jecampagne might have suggestions/opinions here

Reply:
Happy to have suggestions on this. I spent a lot of time working on this specific function and this is the only version I could come up with that is differentiable and doesn't require excess recompilation.

Reply:
Well, if I've fight myself to try to implement a wrapping for the case used by drawFFT, I had no success to get working a   reverse-mode differentiable version due to loop/while. Now, this wrapping is the case of hermx=hermy=False in the Galsim Image.cpp:wrapImage language. if @beckermr 1) gets it right wrt to Galsim, 2) manages to get jacrev ok then it is perfect. I guess it is working because the for_loop bouds are fixed (ie. 0 or image shape). I've nothing to say. In the code @beckermr  it is mentioned that may be you have tried different way to test 
```python
jnp.abs(ii - i) + jnp.abs(jj - j) != 0, 
```
do you see some drawbacks?
By the way where the wrapping non-hermitian is used?

Reply:
I tried a bunch of more logic like things like ii != i with jnp.all etc and got errors. 

Idk where the code uses real space wrapping. However the api supports it so I guess we should implement it.

Reply:
I should retest the gradients on this one btw. It worked for a previous version but I made a few changes and forgot to retest.

Reply:
> By the way where the wrapping non-hermitian is used?

hermitian='x' is the only thing that is used in practice.  I.e. other than in the unit tests of the wrap function.

The non-hermitian case was easier to code up, so I started with that.  Then once I figured out how to do the 'x' version correctly, it seemed worth doing 'y' too, so I wouldn't have to revisit it later if a case for a hermitian image stored that way ever became relevant.  But all the code that calls this have hermitian='x'.